### DISCUSSION TITLE
ElasticSearch built-in security features

### DISCUSSION THREAD

**Discussion by Tealk** [mastodon/mastodon#18426] [Q&A] [answered]
When executing the command `RAILS_ENV=production bin/tootctl search deploy` I get the following message in the console:
`Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.17/security-minimal-setup.html`

There it is e.g. about assigning a password but according to the documentation there is no variable for a password. How should one proceed here?

**Comment by Gargron** [comment, marked as answer]
There is `ES_USER` and `ES_PASS`

**Reply by Tealk** [reply]
Ah thank you, then this is probably missing in the doku:
https://docs.joinmastodon.org/admin/config/
